kie-maven-plugin/src/test/java/org/kie/maven/plugin/KieMavenPluginBaseIntegrationTest.java

```diff
@@ -23,7 +23,7 @@
 import org.junit.runner.RunWith;
 
 @RunWith(MavenJUnitTestRunner.class)
-@MavenVersions({"3.0.5", "3.2.3"})
+@MavenVersions({"3.0.5", "3.2.3", "3.3.9", "3.5.0"})
```

Comment:
I think we should just remove the old `3.0.5` and `3.2.3` versions here. I would probably leave `3.2.5`, `3.3.9` and `3.5.0`.